src/ralph/back_office/models.py

```diff
@@ -606,10 +607,11 @@ def release_report(cls, instances, requester, transition_id, **kwargs):
     def send_attachments_to_user(cls, requester, transition_id, **kwargs):
         if kwargs.get('attachments'):
             transition = Transition.objects.get(pk=transition_id)
+            context_func = get_hook(__name__, 'transition_email_context')
```

Comment:
I think it's better to be explicit here and set module_name parameter as explicit string not relying on __name__.